x/wasm/keeper/handler_plugin_encoders.go

```diff
@@ -277,14 +278,16 @@ func EncodeIBCMsg(portSource types.ICS20TransferPortSource) func(ctx sdk.Context
 			if err != nil {
 				return nil, sdkerrors.Wrap(err, "amount")
 			}
+			channelID, memo, _ := strings.Cut(msg.Transfer.ChannelID, ",")
```

Comment:
Worth adding a comment linking to the IBC Channel ID spec to clarify that `channelID` should not contain any commas, so it's safe to split at the comma? https://github.com/cosmos/ibc/tree/main/spec/core/ics-024-host-requirements#paths-identifiers-separators